server/server.js

```diff
@@ -245,13 +282,19 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
       (this.subs[to] ? 'is subscribed' : 'not subscribed')
     );
 
-    this._persistClientData(socket, message);
-    this._storeResource(resource);
-    this._persistenceSubscribe(resource.to, socket.id);
-    this._updateLimits(socket, message, resource.options);
-    this._stampMessage(socket, message);
-    resource.handleMessage(socket, message);
-    this.emit(message.op, socket, message);
+    this._runMiddleware('post', socket, message, messageType, function (err) {
```

Comment:
"Post" really does not mean anything here, is what I am saying. updateLimits happens to sit at the right spot for this definition of "post". Post what?